src/Catty/ViewController/Stage/StagePresenterSideMenuView.swift

```diff
@@ -144,42 +148,36 @@ enum SideMenuButtonType {
 
         let screenshotButton = setupButton(imageName: "stage_dialog_button_screenshot", selector: #selector(delegate?.takeScreenshotAction), type: .landscapeSecondColumn)
-        screenshotButton.centerYAnchor.constraint(equalTo: self.centerYAnchor, constant: (marginTopBottom - marginLabel) * -1).isActive = true
-
-        let screenshotLabel = setupLabel(title: kLocalizedPreview, selector: #selector(delegate?.takeScreenshotAction), type: .landscapeSecondColumn)
-        screenshotLabel.topAnchor.constraint(equalTo: screenshotButton.bottomAnchor, constant: marginLabel).isActive = true
+        screenshotButton.topAnchor.constraint(equalTo: restartLabel.bottomAnchor, constant: marginLabel * 5).isActive = true
+        screenshotButton.leftAnchor.constraint(equalTo: self.leftAnchor, constant: marginTopBottom * 1).isActive = true
+        screenshotButton.rightAnchor.constraint(equalTo: self.centerXAnchor, constant: marginLabel).isActive = true
 
-        let axesLabel = setupLabel(title: kLocalizedAxes, selector: #selector(delegate?.showHideAxisAction), type: .landscapeFirstColumn)
-        axesLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: (marginTopBottom - marginLabel) * -1).isActive = true
-
-        let axesButton = setupButton(imageName: "stage_dialog_button_toggle_axis", selector: #selector(delegate?.showHideAxisAction), type: .landscapeFirstColumn)
-        axesButton.bottomAnchor.constraint(equalTo: axesLabel.topAnchor, constant: marginLabel * -1).isActive = true
-
-        let aspectRatioLabel = setupLabel(title: kLocalizedMaximize, selector: #selector(self.aspectRatioAction), target: self, type: .landscapeSecondColumn)
-        aspectRatioLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: (marginTopBottom - marginLabel) * -1).isActive = true
-        self.aspectRatioLabel = aspectRatioLabel
+        let axesButton = setupButton(imageName: "stage_dialog_button_toggle_axis", selector: #selector(delegate?.showHideAxisAction), type: .landscapeSecondColumn)
+        axesButton.topAnchor.constraint(equalTo: restartLabel.bottomAnchor, constant: marginLabel * 5).isActive = true
+        axesButton.leftAnchor.constraint(equalTo: self.centerXAnchor, constant: marginLabel * -1).isActive = true
 
         let aspectRatioButton = setupButton(imageName: "stage_dialog_button_aspect_ratio", selector: #selector(self.aspectRatioAction), target: self, type: .landscapeSecondColumn)
-        aspectRatioButton.bottomAnchor.constraint(equalTo: aspectRatioLabel.topAnchor, constant: marginLabel * -1).isActive = true
+        aspectRatioButton.topAnchor.constraint(equalTo: restartLabel.bottomAnchor, constant: marginLabel * 5).isActive = true
+        aspectRatioButton.rightAnchor.constraint(equalTo: self.rightAnchor, constant: marginTopBottom * -1).isActive = true
+        aspectRatioButton.leftAnchor.constraint(equalTo: self.centerXAnchor).isActive = true
```

Comment:
please set the left anchor to the rightAnchor of the Axes Button, otherwise the clickable box is way to big and overlaps with the AxesButton